specifyweb/specify/urls.py

```diff
@@ -30,6 +30,7 @@
         url(r'^(?P<id>\d+)/path/$', tree_views.path),
         url(r'^(?P<id>\d+)/merge/$', tree_views.merge),
         url(r'^(?P<id>\d+)/move/$', tree_views.move),
+        url(r'^(?P<id>\d+)/bulkMove/$', tree_views.bulk_move),
```

Comment:
@melton-jason, 
Would it make sense here to have `r'^(?P<id>\d+)/bulk_move/$'` ? But then we would need to change the frontend bulkMove too...

Reply:
Yes, most other endpoints on the backend use underscores as separators for the names in the url path rather than camelCase. 
`r'^(?P<id>\d+)/bulk_move/$'` matches the consistency used for other endpoints. 

You would need to change the frontend as well: 

https://github.com/specify/specify7/blob/74b5c9a5d9759ca590f3869368a21128ba7d2f99/specifyweb/frontend/js_src/lib/components/TreeView/Actions.tsx#L27-L34

https://github.com/specify/specify7/blob/74b5c9a5d9759ca590f3869368a21128ba7d2f99/specifyweb/frontend/js_src/lib/components/TreeView/Actions.tsx#L306-L309

https://github.com/specify/specify7/blob/74b5c9a5d9759ca590f3869368a21128ba7d2f99/specifyweb/frontend/js_src/lib/components/TreeView/Actions.tsx#L338-L339

https://github.com/specify/specify7/blob/74b5c9a5d9759ca590f3869368a21128ba7d2f99/specifyweb/frontend/js_src/lib/components/TreeView/Actions.tsx#L381-L382

https://github.com/specify/specify7/blob/74b5c9a5d9759ca590f3869368a21128ba7d2f99/specifyweb/frontend/js_src/lib/components/TreeView/Actions.tsx#L421-L422

https://github.com/specify/specify7/blob/74b5c9a5d9759ca590f3869368a21128ba7d2f99/specifyweb/frontend/js_src/lib/components/TreeView/Actions.tsx#L434-L435

https://github.com/specify/specify7/blob/74b5c9a5d9759ca590f3869368a21128ba7d2f99/specifyweb/frontend/js_src/lib/components/TreeView/Actions.tsx#L450-L455

Alternatively, since the frontend prefers camelCase, you can create an Object/function which would map the frontend action name to the backend name and use the mapping before calling the endpoint:

https://github.com/specify/specify7/blob/74b5c9a5d9759ca590f3869368a21128ba7d2f99/specifyweb/frontend/js_src/lib/components/TreeView/Actions.tsx#L319-L330

Reply:
great thank you will do that